src/stores/room-list/algorithms/tag-sorting/RecentAlgorithm.ts

```diff
@@ -20,6 +20,25 @@ import { IAlgorithm } from "./IAlgorithm";
 import { MatrixClientPeg } from "../../../../MatrixClientPeg";
 import * as Unread from "../../../../Unread";
 import { EffectiveMembership, getEffectiveMembership } from "../../../../utils/membership";
+import { EventType } from "matrix-js-sdk/src/@types/event";
+import { MatrixEvent } from "matrix-js-sdk/src/models/event";
+
+const ignoreSelfEvent = (event: MatrixEvent): boolean => {
```

Comment:
As a super minor point: we try to avoid using double negatives in the code where possible. The function name itself is fine, though when used with a `not` like later on it becomes a double negative. Thus, we'd probably name this something like `isNoOpEvent`.